class/Validator.js

```diff
@@ -0,0 +1,59 @@
+let {isEmail} = require('./Helper');
+module.exports = {
+
+    /**
+     * Method of calling functions according to the rules applied 
+     * @param {Object} values - Values to check
+     * @param {Object} arrayCheck - object with the different rules and their messages
+     */
+    check(values, arrayCheck){
+        let errors;
+        Object.keys(values).forEach(key => {
+            
+            const value = values[key];
+            const rules = arrayCheck[key].rules;
+            const messages = arrayCheck[key].messages;
+            let variables = null;
+
+            errors = rules.reduce((acc, rule, index) => {
+
+                if(rule.split(':').length > 1){
```

Comment:
use destructuring to get the same result.
```js
const [ruleName, ...variables] = rule.split(':');
```
variables will default to an empty array.